src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -60,11 +60,14 @@ public JobHostConfiguration(string dashboardAndStorageConnectionString)
             // add our built in services here
             IExtensionRegistry extensions = new DefaultExtensionRegistry();
             ITypeLocator typeLocator = new DefaultTypeLocator(ConsoleProvider.Out, extensions);
+            IConverterManager converterManager = new ConverterManager();
+
             AddService<IExtensionRegistry>(extensions);
             AddService<StorageClientFactory>(new StorageClientFactory());
             AddService<INameResolver>(new DefaultNameResolver());
             AddService<IJobActivator>(DefaultJobActivator.Instance);
             AddService<ITypeLocator>(typeLocator);
+            AddService<IConverterManager>(converterManager);
```

Comment:
If you haven't, add a test to GetService_ReturnsExpectedDefaultServices for this